native/cocos/bindings/manual/jsb_conversions_spec.cpp

```diff
@@ -42,6 +43,7 @@
 #include "scene/Fog.h"
 #include "scene/Shadow.h"
 #include "scene/Skybox.h"
+#include "v8/Object.h"
```

Comment:
Don't invoke v8 specific header.
it's better to use `#include "bindings/jswrapper/Object.h"`

Reply:
fixed